package.lua

```diff
@@ -2,7 +2,7 @@ return {
   name = "rackspace-monitoring-agent",
   version = "2.6.23",
   luvi = {
-    version = "2.9.4-sigar",
+    version = "2.7.6-2-sigar",
```

Comment:
What does this have to do with this change? It's not mentioned in your PR or commit message.

Reply:
The correct sigar version is added to this PR, because the build was failing while looking for 2.9.4, a JAR for which doesn't exist

Reply:
I see an older open PR as well from Richa (correcting this exact same issue)

Reply:
Don't put unrelated changes in the same PR or commit. It should be it's own change, and if there's already a PR for the problem, let's get it merged.